classes/navigationMenu/NavigationMenuDAO.php

```diff
@@ -197,7 +198,7 @@ public function installSettings(int $contextId, string $filename): bool
 
         foreach ($tree->getChildren() as $navigationMenuNode) {
             $site = $navigationMenuNode->getAttribute('site');
-            if ($contextId == \PKP\core\PKPApplication::CONTEXT_ID_NONE && !$site) {
+            if ($contextId === Application::SITE_CONTEXT_ID && !$site) {
```

Comment:
I don't suggest adding the strict checks here unless you're sure they won't run afoul of something like https://github.com/pkp/pkp-lib/issues/10034.

Reply:
Yeah, I'll drop the strict check 😪

Reply:
Even though in this case it might be ok (I guess null values will be retrieved as null instead of an empty string).